### PR TITLE
Issue #79 animator view

### PR DESCRIPTION
**Pull request by mmackiewicz97**
Create a new look of single simulation page, added sorting to animations table

### CODE REVIEW

**Review by AleksanderKamienski — CHANGES_REQUESTED** [review]

File: gui/animator/anim.php [hunk 1]
```diff
@@ -36,8 +36,48 @@ function site() {/*{{{*/
...
+	</tr>
+    </thead>
+    <tbody>
+	<tr>
+	<td>" . number_format($wcbe['0'], 0, ".", "") . " </td>
+	<td>" . number_format($data[0]['dcbe_time'], 0, ".", "") . " </td>
+	<td>" . number_format($results['individual'], 4, ".", "") . " </td>
+	</tr></tbody></table>
+	<table style='width:400px;'><thead><tr>
+	<th>WRI</th>
```
[the remark this answers is not in the document]
Comment (kowalskiw) [reply gui/animator/anim.php:73]: Thank you, the key should be "societal" - my bad.

[the remark this answers is not in the document]
Comment (kowalskiw) [reply gui/animator/anim.php:74]: Fixed in https://github.com/aamks/aamks/pull/156/commits/04a47d256105881a3bb261379d8e4374fdcd99c7

**Review by kowalskiw — APPROVED** [review]
Works fine, to bo developed and improved in the future.

**Review by AleksanderKamienski — APPROVED** [review]

**Comment by kowalskiw** [comment]
Separate button for plotting in the background to be considered before merge.

**Pull request merged**